Neos.Neos/Classes/Routing/FrontendNodeRoutePartHandler.php

```diff
@@ -112,7 +133,7 @@ protected function findValueToMatch($requestPath)
      * in time the route part handler is invoked, the security framework is not yet fully initialized.
      *
      * @param string $requestPath The request path (without leading "/", relative to the current Site Node)
-     * @return boolean true if the $requestPath could be matched, otherwise false
+     * @return bool|MatchResult
```

Comment:
I guess this (the bool for false) is for backwards compatibility? I would otherwise see something like an EmptyMatchResult for that case in the future?

Reply:
Yes for b/c but not related to this change but to the original feature in Flow (https://github.com/neos/flow-development-collection/pull/1126). I added a doc comment to make it clearer